dogpile/cache/region.py

```diff
@@ -327,6 +328,10 @@ def generate_keys(*args):
      by the backend.  Defaults to ``None``, in which case the
      deserializer recommended by the backend will be used.   Typical
      deserializers include ``pickle.dumps`` and ``json.dumps``.
+     Deserializers can raise a :class:`.api.CantDeserializeException` if they
```

Comment:
**mike bayer** (zzzeek) wrote:

make this a separate paragraph.  

then we need to update the versionadded like this:


    .. versionadded:: 1.1.0 added "deserializer" parameter
    .. versionadded:: 1.2.0 added support for :class:`.api.CantDeserializeException`

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/4569